mm/2s2h/Enhancements/Enhancements.cpp

```diff
@@ -29,6 +29,7 @@ void InitEnhancements() {
 
     // Graphics
     RegisterDisableBlackBars();
+    RegisterHWStyledLink();
```

Comment:
I think it'd be easier to understand what this is doing if we write it out in full
```suggestion
    RegisterHyruleWarriorsStyledLink();
```

Reply:
Oh yeah, absolutely, just old habits still sticking around haha